core/src/components/card/card.tsx

```diff
@@ -75,37 +73,47 @@ export class TdsCard {
     <div>
       {this.headerPlacement === 'above' && (
         <div class={`card-top ${this.headerPlacement}`}>
-          {this.headerImg && (
-            <img class={`card-top-image`} src={this.headerImg} alt={this.headerImgAlt} />
-          )}{' '}
+          {this.hasCardThumbnailSlot && (
+            <div class={this.hasCardThumbnailSlot ? 'card-thumbnail' : 'no-header-img'}>
+              <slot name="card-thumbnail"></slot>
+            </div>
+          )}
           <div
             class={`
           card-top-header
-          ${!this.headerImg ? 'no-header-img' : ''}
-          ${!this.header || !this.subheader ? 'single-line-header' : ''}
+          ${!this.hasCardThumbnailSlot ? 'no-header-img' : ''}
+          ${!this.header || !this.hasSubheaderSlot ? 'single-line-header' : ''}
           `}
           >
             <span class={`card-header`}>{this.header}</span>
-            <span class={`card-subheader`}>{this.subheader}</span>
+            {this.hasSubheaderSlot && (
+              <span class={`card-subheader`}>
+                <slot name="card-subheader"></slot>
+              </span>
+            )}
           </div>
         </div>
       )}
       <div class={`card-body`}>
         {this.bodyImg && <img class={`card-body-img`} src={this.bodyImg} alt={this.bodyImgAlt} />}
-        {this.headerPlacement === 'below' && (this.headerImg || this.header || this.subheader) && (
+        {this.headerPlacement === 'below' && (this.header || this.hasSubheaderSlot) && (
           <div class={`card-top ${this.headerPlacement}`}>
-            {this.headerImg && (
-              <img class={`card-top-image`} src={this.headerImg} alt={this.headerImgAlt} />
+            {this.hasCardThumbnailSlot && (
+              <div class={this.hasCardThumbnailSlot ? 'card-thumbnail' : 'no-header-img'}>
```

Comment:
```suggestion
              <div class='card-thumbnail'>
```
Because it will always be this value anyway because of the conditional rendering.

Reply:
🤦 I'll make sure we dont need the CSS first